lib/capistrano/ec2/capistrano_monkey_patch.rb

```diff
@@ -3,7 +3,8 @@
 module Capistrano
   class Configuration
     def for_each_ec2_server(ec2_env:, ec2_role:, &block)
-      ec2 = Fog::Compute.new(provider: 'AWS', region: fetch(:region))
+      ec2 = Fog::Compute.new(provider: 'AWS', region: fetch(:region),
```

Comment:
For the sake of styling, lets comply with out rH code style:

```
Fog::Compute.new \
  provider: 'AWS',
  region: fetch(:region),
  use_iam_profile: fetch(:use_iam_profile, false)
```

Or align them with 2 spaces instead of aligning them on the arguments.